composer.json

```diff
@@ -36,7 +36,7 @@
         "symfony/config": "^4.4 || ^5.4 || ^6.2",
         "symfony/console": "^4.4 || ^5.4 || ^6.2",
         "symfony/dependency-injection": "^4.4 || ^5.4 || ^6.2",
-        "symfony/doctrine-bridge": "^4.4 || ^5.4 || ^6.2",
+        "symfony/doctrine-bridge": "^4.4 <4.4.42 || ^5.4 || ^6.2",
```

Comment:
From 4.4.42 my tests no longer works (it works for 5.4 and 6.2):

there are 3 PRs that affect this release:
https://github.com/symfony/symfony/pull/46443
https://github.com/symfony/symfony/pull/46098
https://github.com/symfony/symfony/pull/46282

I will try to investigate which one is make tests failing... For now, what I see is that with 4.4.42 I get an empty ArrayCollection directly from the EntityType.

Reply:
https://github.com/symfony/symfony/pull/46282 This one is making test fail.

If I manually revert on vendor code, it starts working. Maybe this condition depends on the ORM version? ping @derrabus

Reply:
It is strange that it works with prefer-lowest, 5.4, 6.2, but not with 4.4 (higher deps but fixed to 4.4). Will debug more to see if it is fixable on our side, but any help is welcome.

Reply:
I don't know if I can be of any help here. I'm pretty sure that the change you've linked is correct.

That being said, Symfony 4 is dead. Delivering features for an EOL Symfony version does not seem like a sane thing to do. Hunting down broken tests for an EOL Symfony version seems like a waste of time to me.

Reply:
Solved.